recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
+c_stdlib_version:
+  - "2.28"                 # [linux]
```

Comment:
```suggestion
c_stdlib_version:          # [linux and aarch64]
  - "2.28"                 # [linux and aarch64]
```

Reply:
Will let the current build finish first before commiting this. But just to ask, does the linux64 + CUDA 11.8 builds work with the default c_stdlib_version=2.17 only? Could we not just use c_stdlib_version=2.28 for all builds?

Reply:
GLIBC 2.17 works for everything except CUDA ARM and Power atm

The downside of bumping the GLIBC during the build is this effectively sets the minimum GLIBC supported by the packages produced here. Users won't be able to use them on an older GLIBC. So in general setting the GLIBC as low as possible ensures the greatest compatibility in the packages produced

Hence the suggestion to add this only where it is truly needed

Reply:
Ok, looks like the linux-aarch64 build succeeded :tada: I'm thinking if we could do a build for `linux-aarch64 + CUDA 12.0` across five Python versions (Python 3.9 - 3.13), without re-building the other `linux-64` variants (since there's no changes) to conserve CI resources?

Reply:
I suggest rerendering with all enabled on your local machine to see if any channel pinnings have changed. I think they have because we updated the nvcc pinnings to 12.6 this week. If there are no changes to the .ci_support for the x86 builds, then you can rerender again to skip them. Otherwise, I think it's worth it to rebuild all in order to keep the two builds dependencies synchronized.

Reply:
Yep, cuda_compiler_version has changed from 12.0 to 12.6, and docker_image changed from cos7 to alma9.

```diff
*** linux_64_c_compiler_version12cuda_compilercuda-nvcccuda_compiler_version12.0cxx_compiler_version12python3.10.____cpython.yaml	2024-11-26 11:44:13.902903716 +1300
--- linux_64_c_compiler_version12cuda_compilercuda-nvcccuda_compiler_version12.6cxx_compiler_version12python3.10.____cpython.yaml	2024-11-26 11:41:53.134210447 +1300
***************
*** 17,19 ****
  cuda_compiler_version:
! - '12.0'
  cxx_compiler:
--- 17,19 ----
  cuda_compiler_version:
! - '12.6'
  cxx_compiler:
***************
*** 23,25 ****
  docker_image:
! - quay.io/condaforge/linux-anvil-cos7-x86_64
  github_actions_labels:
--- 23,25 ----
  docker_image:
! - quay.io/condaforge/linux-anvil-x86_64:alma9
  github_actions_labels:
***************
*** 41,43 ****
    - cxx_compiler_version
-   - c_stdlib_version
    - cuda_compiler
--- 41,42 ----
***************
*** 45,46 ****
    - docker_image
-
--- 44 ----
```
will go ahead and re-build :fire: then.

Reply:
Actually, I do suppose we can skip the CUDA 11.8 builds?

Reply:
Sounds good. Thanks!